server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -4327,7 +4327,8 @@ protected long configureCustomRootDiskSize(Map<String, String> customParameters,
      * @throws InvalidParameterValueException if the hypervisor does not support rootdisksize override
      */
     protected void verifyIfHypervisorSupportsRootdiskSizeOverride(HypervisorType hypervisorType) {
-        if (!(hypervisorType == HypervisorType.KVM || hypervisorType == HypervisorType.XenServer || hypervisorType == HypervisorType.VMware || hypervisorType == HypervisorType.Simulator)) {
+        if (!(hypervisorType == HypervisorType.KVM || hypervisorType == HypervisorType.XenServer ||hypervisorType == HypervisorType.VMware ||
+                hypervisorType == HypervisorType.Simulator || hypervisorType == HypervisorType.Custom)) {
```

Comment:
how about `Arrays.asList(HypervisorType.KVM, HypervisorType.XenServer, HypervisorType.VMware, HypervisorType.Simulator, HypervisorType.Custom).contains(hypervisorType)` and even better, mover the array to `Hypervisor.HypervisorType`